omod/src/main/webapp/pages/allergy.gsp

```diff
@@ -60,11 +60,11 @@ ${ ui.includeFragment("allergyui", "removeAllergyDialog") }
             <div id="allergens">
 
                 <div id="types" class="button-group horizontal">
-                    <% allergenTypes.each { category -> %>
+                    <% allergenTypes.each { category ->  if (category.name()!="ENVIRONMENT") { %>
```

Comment:
@dkayiwa Yes basically it's allergens show up under the "other" tab. I had attached the screenshots here already https://issues.openmrs.org/browse/RA-1283